api/app/signals/apps/dsl/evaluators/terminal_evaludator.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+from signals.apps.dsl.evaluators.evaluator import Evaluator
+
+
+class TerminalEvaluator(Evaluator):
+    id_val = None
+    str_val = None
+    numeric_val = None
+    time_val = None
+
+    def __init__(self, **kwargs):
+        self.id_val = kwargs.get('id_val', None)
+        self.str_val = kwargs.get('str_val', None)
+        self.numeric_val = kwargs.get('numeric_val', None)
+        self.time_val = kwargs.get('time_val', None)
+
+    def _convert(self, s, flist=['%H:%M:%S', '%H:%M']):
```

Comment:
not a blocker: but why have `flist=['%H:%M:%S', '%H:%M']` as a parameter, why not self.time_formats or something like this? This is only ever called from within the `TerminalEvaluator`.